app/javascript/components/form/components/fields.jsx

```diff
@@ -23,7 +23,7 @@ const Fields = ({ fields, checkErrors, disableUnderline, formSection, css, formM
       const formUniqueId = formSection?.unique_id || field?.unique_id;
 
       return (
-        <div key={`${formUniqueId}-row`} className={calculatedClasses(field)}>
+        <div key={`${formUniqueId}-row`} className={calculatedClasses(field)} data-testid="form-field">
```

Comment:
Please change to: `form-fields`, because this will render a bunch of fields and not a single one.